### ISSUE TITLE
NTR: [initiator methionine aminopeptidase activity] (the great modification review)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#22698]
* **Suggested term label:**
GO:XXXXXX    initiator methionine aminopeotidase activity

* **Definition (free text)**
Catalysis of the hydrolysis of an initiator methionine or formylmethionine residue from a polypeptide chain.

* **Reference, in format PMID:#######**

PMID:11811952 
PMID:20363127
PMID:20038112
PMID:3027045
(from existing process annotations)

* **Gene product name and ID to be annotated to this term**
see
https://www.ebi.ac.uk/QuickGO/annotations?goUsage=descendants&goUsageRelationships=is_a,part_of,occurs_in&goId=GO:0070084&evidenceCode=ECO:0000269&evidenceCodeUsage=descendants

* **Parent term(s)**

GO:0070006    metalloaminopeptidase activity
or aminopeptidase activity (GO:0004177)
(these appear to be metalloaminopeopidises, but I don't know why we capture the 'metallo' part as it does not  change the reaction...

* **Synonyms (please specify, EXACT, BROAD, NARROW or RELATED)**
related  protein initiator methionine removal 

* **Any other information**

this is to replace the existing process term which should be obsoleted (5 EXP annotations)
 
GO:0035551 | protein initiator methionine removal involved in protein maturation
Removal of the initiating methionine or formylmethionine residue from a protein that contributes to protein maturation, the attainment of the full functional capacity of a protein.

and
protein initiator methionine removal (GO:0070084)
Definition 
The protein modification process in which the translation-initiating methionine or formylmethionine residue is removed from a protein.

**Comment by ValWood** [comment]
Will need this to replace annotation in the protein modification branch

**Comment by ValWood** [comment]
THE EC number for this is EC:3.4.11.18  
Example
https://www.uniprot.org/uniprotkb/P53582/entry

I'm also not sure how it differs from:
Acylamino-acid-releasing enzyme or
acylpeptide hydrolase  (EC 3.4.19.1)
Example:
https://www.uniprot.org/uniprotkb/P13798/entry


@hdrabkin 
- I tried lots of searches to find the GO terms for these 2 EC numbers. Is there a way to search GO by EC?
- Are these terms mapped to RHEA? that might help me to locate them..

**Comment by ValWood** [comment]
@hdrabkin  this ticket was not assigned to anyone. It sound like the sort of thing you would know so I assigned to you.

**Comment by hdrabkin** [comment]
If the EC is associated with a GO term as an xref, using protege one can find it. 
EC:3.4.11.18 (methionyl aminopeptidase.)  is associated with an obsolete term, methionyl aminopeptidase activity, GO:0004239
EC def 'Release of N-terminal amino acids, preferentially methionine, from peptides and arylamides"
methionine aminopeptidase.
peptidase M.
Obsoleted because it represents a gene product. There is no RHEA reaction 

EC:3.4.19.1, acylaminoacyl-peptidase, cleavage of an N-acetyl or N-formyl amino acid from the N-terminus of a polypeptide; NO rhea
Associated with GO:0004252, serine-type endopeptidase activity, 'Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a catalytic mechanism that involves a catalytic triad consisting of a serine nucleophile that is activated by a proton relay involving an acidic residue (e.g. aspartate or glutamate) and a basic residue (usually histidine)."
Note the EC is hydrolysis of internal alpha peptide bonds.
This is a bad association (EC says N-teminus, GO says internal.

 this EC used to be associated with the obsoleted GO term . acylaminoacyl-peptidase activity, GO:0004254.  There is no comment as to why it was obsoleted.

**Comment by ValWood** [comment]
To me, it seems these activities should be represented in GO. 
removing the N-term amino acid is commonly required for protein maturation, and  I don't know how else we could capture  this ?

@pgaudet  what do you think?

**Comment by deustp01** [comment]
> these appear to be metalloaminopeopidises, but I don't know why we capture the 'metallo' part as it does not change the reaction

It does change the reaction mechanism. This was the strategy for the re-organization of peptidase terms: distinguish by mechanism (metallopeptidase, serine peptidase, etc.) and independently by cleavage site: removal of aminoterminal residue of substrate, removal of carboxyterminal residue of substrate, cleavage of a peptide bond that is neither animo- nor carboxyterminal.
Which leads to a kind of nomenclature question

> EC:3.4.19.1, acylaminoacyl-peptidase, cleavage of an N-acetyl or N-formyl amino 

So does removal of an N-conjugated but otherwise aminoterminal amino acid residue count as aminopeptidase because the first amino acid residue is indeed removed, or an an endopeptidase because that removal requires cleavge of the second peptide bond?

**Comment by hdrabkin** [comment]
EC:3.4.19.- vs 3.4.11.-
EC:3.4.19.- Hydrolases  Acting on peptide bonds (peptidases);Omega peptidases
3.4.11.- Hydrolases  Acting on peptide bonds (peptidases);Aminopeptidases

**Comment by hdrabkin** [comment]
I kind of agree that GO:XXXXXX initiator methionine aminopeptidase activity should not have been obsoleted. 
maybe 'N-terminal methionyl aminopeptidase activity is less gene producty? (note new word coined)

**Comment by deustp01** [comment]
> EC:3.4.19.- vs 3.4.11.-

Possibly relevant historical point. At the time that peptidases were reclassified by cleavage site and mechanism, people pointed out that the new classification did not align properly with the EC classification for kinds of peptide bond cleavage, and the GO group decided not to be bothered by the lack of alignment.

**Comment by hdrabkin** [comment]
@deustp01 ; your memory much better than mine;

**Comment by ValWood** [comment]
I remember Midori aligning with Merops. Merops is an excellent resource, but this classification seems a bit orthogonal to GO and often result in us having multiple unconnected MF terms to describe a single protein (i.e endopeptidase, serine-type peptidase, metallo peptidase). To me these don't tell us much about that the  protein is actually _doing_ Personally I think it would be better t for our purposes to align proteases with Rhea.

**Comment by hdrabkin** [comment]
I'm not sure RHEA would be a good pick because basically all the reactions would be the same (hydrolysis of a peptide bond). 
Let's stay on track.
1. the two reactions mentioned at the start are not the same
2. How/should we bring back the initiator methionine aminopeptidase activity; regardless of how it works, its unique specificity should be in GO to represent a specific protein processing step.

**Comment by pgaudet** [comment]
I agree we should restore/recreate GO:0004239 'obsolete methionyl aminopeptidase activity'
with EC xef  EC:3.4.11.18

@cmungall is it OK for you if we restore obsolete terms?

**Comment by ValWood** [comment]
ping 
@cmungall is it OK for you if we restore obsolete terms?

**Comment by ValWood** [comment]
ping
@cmungall is it OK for you if we restore obsolete terms?

**Comment by pgaudet** [comment]
Sure let's restore, we have done that before.

**Comment by pgaudet** [comment]
https://wiki.geneontology.org/Restoring_an_Obsolete_Ontology_Term
